packages/gatsby-telemetry/src/event-storage.js

```diff
@@ -10,20 +14,52 @@ const isTruthy = require(`./is-truthy`)
  * to continue even when working offline.
  */
 module.exports = class EventStorage {
-  store = new Store()
-  debugEvents = isTruthy(process.env.GATSBY_TELEMETRY_DEBUG)
-  disabled = isTruthy(process.env.GATSBY_TELEMETRY_DISABLED)
+  constructor() {
+    try {
+      this.config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+    } catch (e) {
+      // This should never happen
+      this.config = {
+        get: key => this.config[key],
+        set: (key, value) => (this.config[key] = value),
+        all: this.config,
+        path: path.join(os.tmpdir(), `gatsby`),
+        "telemetry.enabled": true,
+        "telemetry.machineId": `not-a-machine-id`,
+      }
+    }
+
+    const baseDir = path.dirname(this.config.path)
+
+    try {
+      ensureDirSync(baseDir)
+    } catch (e) {
+      // TODO: Log this event
+    }
+
+    this.store = new Store(baseDir)
+    this.verbose = isTruthy(process.env.GATSBY_TELEMETRY_VERBOSE)
+    this.debugEvents = isTruthy(process.env.GATSBY_TELEMETRY_DEBUG)
+    this.disabled = isTruthy(process.env.GATSBY_TELEMETRY_DISABLED)
+  }
 
   addEvent(event) {
     if (this.disabled) {
       return
     }
 
-    if (this.debugEvents) {
-      console.error(`Captured event:`, JSON.stringify(event))
-    } else {
-      this.store.appendToBuffer(JSON.stringify(event) + `\n`)
+    const eventString = JSON.stringify(event)
+
+    if (this.debugEvents || this.verbose) {
+      console.error(`Captured event:`, eventString)
```

Comment:
@jamo why is this an error? Shouldn't this be something like info? we have reporter.info or console.info. Could you change this in a next iteration?